koku/api/settings/tags/mapping/view.py

```diff
@@ -129,7 +130,7 @@ def put(self, request):
         children_rows = list(EnabledTagKeys.objects.filter(uuid__in=serializer.data.get("children")))
         tag_mappings = [TagMapping(parent=parent_row, child=child_row) for child_row in children_rows]
         TagMapping.objects.bulk_create(tag_mappings)
-        resummarize_current_month_by_tag_keys(children_rows, request.user.customer.schema_name)
+        resummarize_current_month_by_tag_keys(serializer.data.get("children"), request.user.customer.schema_name)
```

Comment:
A `TypeError` will be raised if `None` is passed to the function.
```suggestion
        resummarize_current_month_by_tag_keys(serializer.data.get("children", []), request.user.customer.schema_name)
```